harness/src/fuzz/mollusk.rs

```diff
@@ -94,6 +94,7 @@ impl From<&FuzzEffects> for InstructionResult {
             execution_time,
             program_result,
             raw_result,
+            return_data: None, // TODO: Omitted for now.
```

Comment:
Can you add this field to the non-Firedancer protobuf and fixture lib? It should be pretty simple, let's just do `Vec<u8>` like FD does.

Now that the Mollusk result type tracks return data, the Mollusk-based fixture effects should also.